interfaces/interfaces.go

```diff
@@ -2,7 +2,43 @@ package interfaces
 
 type IWritter interface {
 	WriteToDisk() error
+	CheckIfExists() error
+	DeletePath() error
+	GetPath() string
 }
 
 // TODO:
 // add objects that satisfy the interface
+
+func Write(paths []IWritter) error {
+	for _, path := range paths {
+		err := path.WriteToDisk()
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func CheckInterface(paths []IWritter) error {
```

Comment:
suggesting renaming to CheckPathExists or something else without interface in the name

Reply:
addressed in fa38b125828f208631c482c74923ffb3fbe4f1da

or 

https://github.com/TudorHulban/GoLayouter/blob/fa38b125828f208631c482c74923ffb3fbe4f1da/domain/interfaces/interfaces_helpers.go#L43